ibis/mapd/compiler.py

```diff
@@ -241,3 +241,5 @@ def mapd_rewrite_not_any(expr):
         byte_length=_unary_op('length', mapd_ops.ByteLength)
     )
 )
+
+_add_methods(ir.TableExpr, dict(union=mapd_ops._union))
```

Comment:
@cpcloud @kszucs 

it seems it is not the correctly way to do that.

I added this because when union with head, for example:

```
        t1 = alltypes.head()
        t2 = alltypes.head()
        t1.union(t2)
```
it doesn't get the union method from MapDTable. the return of `head` is an ir.TableExpr and it doesn't recognize  the `union` from MapDTable   neither from `_operation_registry`.

do you have any idea to fix that?

Reply:
@kszucs @cpcloud any idea?

Reply:
sure! I am doing that!

Reply:
@cpcloud @kszucs do you have any idea the best way to fix this problem?

Reply:
?

Reply:
Can you show the result of `print(t1.union(t2))` as I requested above?

Reply:
sorry I thought I should add this in the tests: https://github.com/ibis-project/ibis/pull/1654/files#diff-a2b863e1bd07ae9d02e1258e5cffb7cbR68

well actually this doesn't return nothing because It should raise an error:

` UnsupportedOperationError: MapD backend doesn't support Union operation`

this error is from MapDTable.union (https://github.com/ibis-project/ibis/pull/1654/files#diff-d322a905d6bb08b57087c8c0c38f9d32R290)

and `print(type(t1))` return `<class 'ibis.expr.types.TableExpr'>`

Reply:
@cpcloud @kszucs any feedback?